contents/data/sql/SQL.md

```diff
@@ -0,0 +1,145 @@
+<frontmatter>
+  title: Introduction to SQL
+  footer: footer.md
+  head: head.md
+  siteNav: mainNav.md
+  pageNav: 3
+</frontmatter>
+
+{{ navbar | safe }}
+
+<div class="website-content">
+
+# Introduction to SQL
+
+**Author(s): [Amrut Prabhu](https://github.com/amrut-prabhu)**
+
+Reviewers: [Ronak Lakhotia](https://github.com/RonakLakhotia), [Rahul Rajesh](https://github.com/rrtheonlyone)
+
+## What is SQL?
+
+Databases are integral to any commercial software application, whether it is a web app, desktop app or otherwise.
+As the size of your application grows, you need to look into dedicated database software packages (like SQL software) to store your data. It will not be enough to simply rely on in-memory solutions or a simple text file to store the data. Many companies like Facebook, Google, Adobe, Paytm and Zappos use <tooltip content="Relational DataBase Management System">RDBMS</tooltip> software to fulfil this need.
+
+**Structured Query Language** (**SQL**) is a programming language that is specifically designed for managing (storing, retrieving and manipulating) data in a <tooltip content="relational databases store data in tables">[relational database]({{baseUrl}}/contents/data/databases/databases.html#database-models)</tooltip>. Unlike other languages, SQL doesn't come as a standalone installation. Rather, _SQL Software_ (or _SQL Implementations_) like MySQL, Oracle, SQL Server and PostgreSQL come with a RDBMS that uses the SQL language.
+However, the syntaxes and functions used across these software systems <tooltip content="in terms of whether the syntax is case-sensitive, the format for specifying date and time, what functions are available out-of-the-box">vary</tooltip>.
+
+<box type="tip">
+  Many online resources refer to Microsoft SQL Server as SQL. So, keep that in mind before you start making conclusions about the characteristics and features of SQL, the language.
+</box>
+
+---
+
+### How does SQL work?
+
+SQL uses **queries** to retrieve data. Here is an example of how an SQL query is used.
+
+Suppose we have the data table `Students` shown here:
+
+| ID | Name  | Course | Faculty |
+| -- | ----- | ------ | ------- |
+| 1  | Alex  | CS202  | CS      |
+| 2  | Bob   | MA303  | MA      |
+| 3  | Cathy | CS202  | CS      |
+| 4  | Daren | CS202  | CS      |
+| 5  | Ellie | CS101  | CS      |
+| 6  | Fred  | MA303  | MA      |
+| 7  | Gary  | CS101  | CS      |
+| 8  | Henry | CS404  | CS      |
+
+We can use this SQL query to retrieve information from this table:
+```
+SELECT Course, COUNT(*) num
+FROM Students
+WHERE Faculty = 'CS'
+GROUP BY Course
+HAVING COUNT(*) > 1
+ORDER BY num;
+```
+
+<box type="tip">
+  The uppercase words in this query are either **clauses** or **statements**, and are SQL keywords.
```

Comment:
Add a blank line before this line. MarkBind needs blank line at the start of markdown syntax. Otherwise it renders like this:
![image](https://user-images.githubusercontent.com/1673303/54971796-29cc9080-4fc3-11e9-8b2a-7ba891dc9bd6.png)